src/System.Collections.Immutable/tests/System.Collections.Immutable.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{95DFC527-4DC1-495E-97D7-E94EE1F7140D}</ProjectGuid>
-    <NoWarn>0436</NoWarn>
+    <NoWarn>0436;NU5105;xUnit2017</NoWarn>
```

Comment:
This can be reverted.

Reply:
Sure, I was considering it as one of the alternatives for disabling the rule and then it went in by mistake.